### PR TITLE
Try retrying to resolve intermittent failures

### PR DESCRIPTION
**Pull request by olaughter**

### CODE REVIEW

File: src/index/vespa_.py [hunk 1]
```diff
@@ -266,7 +267,7 @@ def _get_vespa_instance() -> Vespa:
         cert=cert_location,
     )
 
-
+@retry(retry=retry_if_exception_type(VespaIndexError), wait=wait_fixed(5), stop=stop_after_attempt(3))
```
Comment (diversemix) [inline src/index/vespa_.py:270]: Can we use exponential backoff? i.e. `wait_exponential` would seem to make more sense to me :smile_cat:

Comment (diversemix) [reply src/index/vespa_.py:270]: Can we also add logging too: https://tenacity.readthedocs.io/en/latest/#before-and-after-retry-and-logging

Comment (olaughter) [reply src/index/vespa_.py]: Updated the former, we do have a log message there already on failure, does this cover what you mean?

Comment (diversemix) [reply src/index/vespa_.py]: yeah - I was just thinking more logging the merrier - I don't know if the logging will give us extra info :shrug: buthappy

**Pull request merged**